### ISSUE TITLE
2nd Monitor (Eizo) not working on iMac 14,2

### ISSUE THREAD

**Issue by BertNase** [dortania/OpenCore-Legacy-Patcher#213]
I switched from micropatcher 0.5.1 to opencore using OCLP 0.1.2 and macOS 11.3. Since
then my external Eizo Monitor connected via Thunderbolt 2 isn't recognized any longer.

I did update to 11.3.1 without problems but monitor doesn't work. I installed macOS 11.3.1 using micropatcher 0.5.1 again to an external drive to check it out and monitor is recognized using micropatcher. I didn't do any post install things.

iMac late 2013 14,2
OS Running macOS 11.3.1 with OLCP 0.1.2

**Comment by Ausdauersportler** [comment]
Read this [issue #190](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/190), possibly you need the same setting.

**Comment by Ausdauersportler** [comment]
@khronokernel
Do you have this vit9696 added to all the iMacs with NVIDIA dGPU? This comes up every day now :-)

**Comment by khronokernel** [comment]
@Ausdauersportler Correct, it’s already in 0.1.3. Seems I should do a new release soon ;p
